robotiq_gazebo/config/gripper_controller.yaml

```diff
@@ -0,0 +1,11 @@
+a_bot_gripper_controller:
```

Comment:
This is specific to our setup and should be in `o2ac_scene_description`. You can leave an example `gripper_controller` in this package though.

Reply:
I moved the controller yaml to o2ac_gazebo instead. That's where the 2_bot_controller used to control the arms in gazebo is as well.